.github/workflows/ci.yaml

```diff
@@ -0,0 +1,279 @@
+              run: |
+                  rm src/Migrations/*
+                  echo "        'Synolia\SyliusSchedulerCommandPlugin\Migrations': '@SynoliaSyliusSchedulerCommandPlugin/Migrations'" >> config/packages/doctrine_migrations.yaml
+              if: matrix.sylius != '1.7.*'
+
+            - name: Behat - Config Suite
+              working-directory: ./sylius
+              run: |
+                  sed -i "s#vendor/sylius/sylius/src/Sylius/Behat/Resources/config/suites.yml#vendor/${{ env.package-name }}/tests/Behat/Resources/suites.yml#g" behat.yml.dist
+                  sed -i "s#vendor/sylius/sylius/features#vendor/${{ env.package-name }}/features#g" behat.yml.dist
+                  sed -i "s#@cli#@javascript#g" behat.yml.dist
+                  echo '    - { resource: "../vendor/${{ env.package-name }}/tests/Behat/Resources/services.yml" }' >> config/services_test.yaml
+
+            - name: Sylius - Set database config
+              working-directory: ./sylius
+              run: |
+                  echo 'DATABASE_URL=mysql://root:@127.0.0.1:3306/scheduler_plugin_%kernel.environment%?serverVersion=mariadb-10.4.11' >> .env
+
+            - name: Sylius - Copy fixtures
+              working-directory: ./sylius
+              run: cp ../plugin/tests/datas/sylius_fixtures.yaml config/packages
+
+            - name: Sylius - Install
+              working-directory: ./sylius
+              run: php bin/console sylius:install -n -s scheduler_command
+
+            - name: Yarn - Get cache directory
+              id: yarn-cache
+              run: echo "::set-output name=dir::$(yarn cache dir)"
+
+            - name: Yarn - Set Cache
+              uses: actions/cache@v2
+              with:
+                  path: ${{ steps.yarn-cache.outputs.dir }}
+                  key: node-${{ matrix.node }}-yarn-${{ hashFiles('**/package.json **/yarn.lock') }}
+                  restore-keys: |
+                      node-${{ matrix.node }}-yarn-
+
+            - name: Sylius - Assets
+              working-directory: ./sylius
+              run: yarn install && yarn build
+
+            - name: Sylius - Cache clear
+              working-directory: ./sylius
+              run: |
+                  php bin/console c:c
+
+            - name: PHPUnit - Configure
+              working-directory: ./sylius
+              run: |
+                  cp vendor/${{ env.package-name }}/phpunit.xml.dist phpunit.xml
+                  sed -i "s#tests/Application/config/bootstrap.php#config/bootstrap.php#g" phpunit.xml
+                  sed -i "s#<directory>tests</directory>#<directory>./vendor/${{ env.package-name }}/tests/PHPUnit</directory>#g" phpunit.xml
+                  sed -i "s#/tests/Application/Kernel.php#src/Kernel.php#g" phpunit.xml
+
+            - name: Output PHP version for Symfony CLI
+              working-directory: ./sylius
+              run: php -v | head -n 1 | awk '{ print $2 }' > .php-version
```

Comment:
Any reason to not use ${{ matrix.php }} ?

Reply:
I don't know enough to give you an answer. I relied on
https://github.com/Sylius/PluginSkeleton/blob/51f078c13afd593a66f3f05acdd055f3e1b20418/.github/workflows/build.yml#L70